docs/development.md

```diff
@@ -0,0 +1,133 @@
+﻿# FrankenPHP Development Environment Setup
+
+---
+
+## For Windows: WSL2 Setup
+
+1. Install WSL2:
+
+   ```powershell
+   wsl --install
+   ```
+
+2. Install a Linux distribution (example: AlmaLinux 10) via e.g. `wsl --install AlmaLinux-10`.
+
+---
+
+## Initial setup
+
+Follow the guide on [compiling from sources](compile.md).
+We will assume you installed these things into the following paths:
```

Comment:
The "We will assume" feels weird. Maybe something like "As of from now, it is assumed that the following tools are already installed". Also, using "these things" doesn't feel appropriate here, feels to oral to me

Reply:
I agree, rewrote it.